### PR TITLE
feat: initial user interface

### PR DESCRIPTION
**Pull request by iuricmp**
- [x] initial screens to manage the GNO integration.
- [x] initial home screen that enables a Reply on an existing Thread.
- [x] initial settings screen that displays the Gno used configuration.
- [x] prompt do display the result

### CODE REVIEW

**Comment by iuricmp** [comment]
I think this is ok for an initial UI. I'm planning to open a new PR with more improvements and fixes for issue we find on it.

**Review by jefft0 — APPROVED** [review]
The app runs for me on simulator.

File: gnoboard/REAME.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Experimental mobile app for GNO
+
+
+## 🗂️ Directory layout
+
+| directory                                            | description                                                                                    |
+| ---------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| [src/native_modules/](./src/native_modules)              | Native interface code                                                                          |
+| [src/components](./src/components)                       | React Native components                                                                        |
+| [src/navigation/](./src/navigation)                      | Berty Messenger + react-native-navigation                                                      |
```
Comment (D4ryl00) [inline gnoboard/REAME.md:12]:
```suggestion
| [src/navigation/](./src/navigation)                      | Gnoboard + react-native-navigation                                                      |
```

**Review by D4ryl00 — APPROVED** [review]
LGTM

**Pull request merged**